aws/sdk/build.gradle.kts

```diff
@@ -369,9 +367,10 @@ tasks.register("finalizeSdk") {
         "relocateExamples",
         "generateIndexMd",
         "fixManifests",
+        "generateVersionManifest",
+        "fixExampleManifests",
         "hydrateReadme",
-        "relocateChangelog",
-        "generateVersionManifest"
```

Comment:
we are removing this because the versions manifest will start being updated differently?

Reply:
It wasn't removed, just reordered. The versions manifest needs to get generated before the example manifests can be fixed now.